examples/step-10/step-10.cc

```diff
@@ -95,17 +95,9 @@ namespace Step10
         std::cout << "Refinement level: " << refinement << std::endl;
 
         // Then have a string which denotes the base part of the names of the
-        // files into which we write the output. Note that in the parentheses
-        // in the initializer we do arithmetic on characters, which assumes
-        // that first the characters denoting numbers are placed consecutively
-        // (which is probably true for all reasonable character sets
-        // nowadays), but also assumes that the increment
-        // <code>refinement</code> is less than ten. This is therefore more a
-        // quick hack if we know exactly the values which the increment can
-        // assume. A better implementation would use the
-        // <code>std::istringstream</code> class to generate a name.
-        std::string filename_base = "ball";
-        filename_base += '0'+refinement;
+        // files into which we write the output (ending with the refinement
+        // level).
+        std::string filename_base = "ball" + Utilities::to_string(refinement);
```

Comment:
Do you feel strongly about this form of the file name, or would you indulge me in using `ball_` as prefix?